### PR TITLE
Add withdrawn notice to html publications

### PR DESCRIPTION
**Pull request by andysellick**
- uses notice component
- requires update to translations, have copied the lines for 'publication' as 'html publication' wouldn't make sense to most people
- updated non-English language files as well, used the translation for 'publication' there as well

screen shot 2017-09-07 at 09 28 48

https://trello.com/c/23VcfUo2/137-1-add-withdrawn-notice-to-html-publications

Examples:
- https://government-frontend-pr-474.herokuapp.com/government/publications/dependants-of-students-and-prospective-students-sty01/dependants-of-students-and-prospective-students-sty01
- https://government-frontend-pr-474.herokuapp.com/government/publications/fees-for-civil-and-family-courts/court-fees-for-the-high-court-county-court-and-family-court
- https://government-frontend-pr-474.herokuapp.com/government/publications/ex35-6ne-environmental-permit-application-advertisement/ex35-6ne-environmental-permit-application-advertisement

### CODE REVIEW

**Comment by nickcolley** [comment]
This looks good but it goes against the pattern of the other document types so I'm not sure. cc @fofr

**Comment by fofr** [comment]
I've a couple of suggestions.

Can we put the withdrawn notice below the title? The title is still appropriate to the content of the withdrawn notice, I fear that putting the notice too high will mean it doesn't have context. ie "This" doesn't have a clear reference.

Regarding translations, the content type shown in the withdrawal notice should match the HTML publication’s parent – eg in the screenshot it would be guidance.

**Comment by gpeng** [comment]
Looks good,. I agree with @fofr on both those points. The parent `document_type` would be more appropriate although we don't translate the rest of that title so it's already a bit weird e.g. https://www.gov.uk/government/publications/compliance-checks-serial-tax-avoidance-warning-notices-ccfs38.cy & https://www.gov.uk/government/publications/russia-economic-gaidar-forum-2015.ru We also don't provide functionality for publishers to add translations of their withdrawal notice. So... if it's complex to do I'm not sure it's really worth the effort without fixing the rest.

File: app/presenters/html_publication_presenter.rb [hunk 1]
```diff
@@ -56,4 +60,8 @@ def full_path(request)
   def public_timestamp
     content_item["details"]["public_timestamp"]
   end
+
+  def withdrawal_notice_context
+    I18n.t("content_item.schema_name.#{format_sub_type}", count: 1) if format_sub_type
```
Comment (fofr) [inline app/presenters/html_publication_presenter.rb]: Can probably remove `if format_sub_type` now

**Comment by fofr** [comment]
Can you update screenshots and link to examples on the review app?

File: app/presenters/html_publication_presenter.rb (hunk 1 above)
Comment (andysellick) [reply app/presenters/html_publication_presenter.rb]: Good point, thanks.

**Comment by andysellick** [comment]
Screenshot updated and links added.

File: app/presenters/html_publication_presenter.rb [hunk 2]
```diff
@@ -20,7 +20,11 @@ def copyright_year
   end
 
   def format_sub_type
```
Comment (fofr) [inline app/presenters/html_publication_presenter.rb]: Unrelated to this PR – this method would be better called "parent_document_type"

**Pull request merged**